frontend/www/js/omegaup/components/user/GridPaginator.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h2 class="panel-title">{{ title }} <span class="badge">{{ problems.length }}</span></h2>
+    </div>
+    <table class="table table-striped"
+           v-if="problems.length &gt; 0">
+      <tbody>
+        <tr v-for="group in paginatedProblems">
+          <td v-for="problem in group">
+            <a v-bind:href="`/arena/problem/${problem.alias}`">{{ problem.title }}</a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <div class="panel-footer"
```

Comment:
```suggestion
    <div class="panel-footer text-center"
```

así puedes eliminar L34-39, cortesía de [Bootstrap](https://getbootstrap.com/docs/3.4/css/#type-alignment)!